test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -152,13 +152,14 @@
 
   private static final String CREATE_TABLE_AUDITTRAIL =
       "create table dm_audittrail "
-      + "(r_object_id varchar, audited_obj_id varchar, chronicle_id varchar, "
-      + "event_name varchar, time_stamp_utc timestamp, attribute_list varchar)";
+      + "(r_object_id varchar unique, audited_obj_id varchar, "
+      + "chronicle_id varchar, event_name varchar, time_stamp_utc timestamp, "
+      + "attribute_list varchar)";
 
   private static final String CREATE_TABLE_AUDITTRAIL_ACL =
       "create table dm_audittrail_acl "
-      + "(r_object_id varchar, chronicle_id varchar, audited_obj_id varchar, "
-      + "event_name varchar, time_stamp_utc timestamp)";
+      + "(r_object_id varchar unique, audited_obj_id varchar, "
+      + "chronicle_id varchar, event_name varchar, time_stamp_utc timestamp)";
 
   private static final String CREATE_TABLE_CABINET = "create table dm_cabinet "
       + "(r_object_id varchar, i_chronicle_id varchar, r_folder_path varchar, "
```

Comment:
The cabinetsWhereCondition is explicitly allowed to make the query against dm_cabinets ROW_BASED, though we don't actually try that in the tests.